### DISCUSSION TITLE
The size of the files created increases when I batch watermark

### DISCUSSION THREAD

**Discussion by hobapli** [ImageMagick/ImageMagick#4939] [Help]
Original file size is about 250 KB , when im batching the avarage the file size is 600kb with watermark. Do you know any idea?

the code is 
@echo off
for  %%z in (*.png ) do (
   
   magick "%%z" logo.png -gravity northeast -geometry +2+2  -composite output/"%%z" 
)

**Comment by fmw42** [comment]
Please provide image examples and identify your IM version and platform.
